tests/foreman/ui/test_organization.py

```diff
@@ -16,16 +16,49 @@
 
 :Upstream: No
 """
+import logging
+
 import pytest
 from fauxfactory import gen_string
 from nailgun import entities
 
 from robottelo.config import settings
 from robottelo.constants import DEFAULT_ORG
+from robottelo.constants import DISTRO_RHEL7
 from robottelo.constants import INSTALL_MEDIUM_URL
 from robottelo.constants import LIBVIRT_RESOURCE_URL
+from robottelo.constants import VALID_GPG_KEY_FILE
+from robottelo.helpers import read_data_file
 from robottelo.manifests import original_manifest
 from robottelo.manifests import upload_manifest_locked
+from robottelo.products import RepositoryCollection
+from robottelo.products import RHELAnsibleEngineRepository
+from robottelo.products import SatelliteToolsRepository
+from robottelo.products import YumRepository
+
+CUSTOM_REPO_URL = settings.repos.yum_6.url
+CUSTOM_REPO_ERRATA_ID = settings.repos.yum_6.errata[2]
+logger = logging.getLogger('robottelo')
+
+
+@pytest.fixture(scope='module')
+def module_repos_col(module_org, module_lce, default_sat, request):
+    default_sat.api.Parameter(
+        name='remote_execution_connect_by_ip', value='Yes', organization=module_org.id
```

Comment:
@shweta83 can you explain why we need to update this setting for tests?

Reply:
This was added for different test which is not yet added. Updated this.